common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -188,3 +188,9 @@ def definition_to_xml(self, resource_fs):
         for child in self.get_children():
             self.runtime.add_block_as_child_node(child, xml_object)
         return xml_object
+
+    @property
+    def non_editable_metadata_fields(self):
```

Comment:
Please add a unit test for seq_module's method, and add a check for is_entrance_exam to the existing test_non_editable_settings in test_split_test_module.py.